apps/style/common.scss

```diff
@@ -1655,11 +1646,11 @@ $see-examples-thick-underline-color: $yellow;
 
 .droplet-block-tooltipster .tooltipster-content {
   font-family: $tooltip-font;
+  font-weight: 400;
```

Comment:
Should we store separate constants for font weight as well?

Reply:
Not sure if we'll be using it often, but yeah, I guess that a good idea, will do, thank you!

Reply:
Added